icu4c/source/tools/toolutil/writesrc.cpp

```diff
@@ -247,38 +249,58 @@ U_CAPI void U_EXPORT2
 usrc_writeUCPTrieArrays(FILE *f,
                         const char *indexPrefix, const char *dataPrefix,
                         const UCPTrie *pTrie,
-                        const char *postfix) {
-    usrc_writeArray(f, indexPrefix, pTrie->index, 16, pTrie->indexLength, postfix);
+                        const char *postfix,
+                        UTargetSyntax syntax) {
+    // "\t" reduces file size relative to "    "
```

Comment:
It does, but it gets displayed inconsistently... Easiest to not indent the often long array initializers. Not super pretty, but seems ok for this sort of data.

Reply:
Discussed and changed to two spaces